astroid/brain/brain_builtin_inference.py

```diff
@@ -570,12 +574,16 @@ def infer_property(node, context=None):
     prop_func = objects.Property(
         function=inferred,
         name=inferred.name,
-        doc=getattr(inferred, "doc", None),
+        doc=getattr(inferred, "doc_node.value", None),
```

Comment:
```suggestion
```
`Property` inherits from `FunctionDef` and the `doc` attribute is only passed to `super().__init__` so it should be safe to remove it now.